Resources/Prototypes/Catalog/uplink_catalog.yml

```
@@ -37,7 +37,7 @@
   discountDownTo:
     Telecrystal: 2
   cost:
-    Telecrystal: 4
+    Telecrystal: 3 # Sunrise-Edit
```

YAMLlint (1.37.1)

[warning] too few spaces before comment: expected 2

(comments)